Makefile

```diff
@@ -554,6 +554,11 @@ run_bpf_tests: ## Build and run the BPF unit tests using the cilium-builder cont
 			"JUNIT_PATH=$(JUNIT_PATH)" \
 			"V=$(BPF_TEST_VERBOSE)"
 
+run_bpf_benchmarks: ## Build and run the BPF benchmarks tests using the cilium-builder container image.
```

Comment:
nit: `bench-bpf` to follow the existing bench prefix for benchmark targets.

Reply:
Sure. I'll also move the target up there to group all the `bench` stuff together.